portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,50 +14,25 @@
 
 package com.google.sps.servlets;
 
-import com.google.gson.Gson;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import java.io.IOException;
 import java.util.ArrayList;
-import java.util.List;
 import javax.servlet.annotation.WebServlet;
-import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /**
  * Servlet that stubs response from /data URL. Expected response is of type String.
  */
 @WebServlet("/data")
-public class DataServlet extends HttpServlet {
+public class DataServlet extends HttpServletWithUtilities {
```

Comment:
In the future, do cleanup like this in a separate PR.

Reply:
Agreed - should've been a PR ahead of this one. Will change for next time!